### ISSUE TITLE
CSP: Remove @babel/runtime from dependencies

### ISSUE THREAD

**Issue by deleonio** [ant-design/ant-design-icons#491]
`@babel/runtime` contains a dependency `regenenerator-runtime`. And this break the Content-Security-Policy `unsafe-eval`.

I think, that `@babel/runtime` should not be a part of dependency. It must be a part of devDependency?!

**Comment by deleonio** [comment]
Read that: @tellnes

No issue tab?! - https://github.com/ant-design/react-slick

**Comment by tellnes** [comment]
> Read that: @tellnes
> 
> No issue tab?! - https://github.com/ant-design/react-slick

?

**Comment by deleonio** [comment]
@tellnes : at the repo - there is no active isssue tab. Please logout and try it.

We have CSP errors and we should remove the `unsafe-eval`.

**Comment by tellnes** [comment]
Yes, but why do you mention me? I'm not a maintainer, just a random contributor sending the latest PR that got merged to `@ant-design/react-slick`.

**Comment by deleonio** [comment]
Fix it with your bundler: https://leanupjs.org/1.2/guide/migration/#if-you-have-unsafe-eval-with-babel-runtime

**Comment by deleonio** [comment]
@yoyo837 - Please check this issue.

**Comment by nicolo-ribaudo** [comment]
Hi! This issue can probably be closed; `@babel/runtime@7.18.0` does not depend on `regenerator-runtime` anymore.
